recipe/meta.yaml

```diff
@@ -24,12 +24,10 @@ source:
 
 build:
   number: {{ build_number }}
+  noarch: python
   # Windows CUDA 11.8 fails due to a compilation error
   # xref: https://github.com/conda-forge/xgboost-feedstock/issues/173
   skip: True  # [win and cuda_compiler_version == "11.8"]
-  {% if python is defined %}
-  skip: {{ python.split(".")[:2] != min_python.split(".")[:2] }}
-  {% endif %}
```

Comment:
Good intuition Hyunsu! 🙏

Think we can fix this up to handle both cases then and fix the linter

Though ok doing this as a follow up